src/core/server/saved_objects/export/sort_objects.ts

```diff
@@ -30,11 +29,7 @@ export function sortObjects(savedObjects: SavedObject[]): SavedObject[] {
   function includeObjects(objects: SavedObject[]) {
     for (const object of objects) {
       if (path.has(object)) {
-        throw Boom.badRequest(
-          `circular reference: ${[...path, object]
-            .map((obj) => `[${obj.type}:${obj.id}]`)
-            .join(' ref-> ')}`
-        );
+        continue;
```

Comment:
Not sure, but can we now delete the ` path.delete(object);` L47?

Reply:
I don't think so. It acts as a pop operation in 'path stack'